pkgs/applications/audio/hivelytracker/default.nix

```
@@ -0,0 +1,76 @@
+{ stdenv
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, SDL
+, SDL_image
+, SDL_ttf
+, gtk2
+, glib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hivelytracker";
+  version = "unstable-2020-08-19";
+
+  src = fetchFromGitHub {
+    owner = "pete-gordon";
+    repo = "hivelytracker";
+    rev = "c8e3c7a5ee9f4a07cb4a941caecf7e4c4f4d40e0";
+    sha256 = "1nqianlf1msir6wqwapi7ys1vbmf6aik58wa54b6cn5v6kwxh75a";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    makeWrapper
+  ];
+
+  buildInputs = [
+    SDL
+    SDL_image
+    SDL_ttf
+    gtk2
+    glib
+  ];
+
+  # Cannot use sourceRoot because we need the whole tree to be writable
+  preBuild = "cd sdl";
+
+  makeFlags = [ "-f Makefile.linux" "PREFIX=$(out)" ];
+
+  NIX_CFLAGS_COMPILE =
+    "-I${SDL}/include/SDL \
+     -I${SDL_image}/include/SDL \
+     -I${SDL_ttf}/include/SDL \
+     -I${gtk2.dev}/include/gtk-2.0 \
+     -I${glib.dev}/include/glib-2.0";
```

I think this'd be a little nicer, assuming the Darwin version doesn't use GTK2.

```suggestion
  NIX_CFLAGS_COMPILE = stdenv.lib.concatStringsSep " " ([
    "-I${SDL}/include/SDL"
    "-I${SDL_image}/include/SDL"
    "-I${SDL_ttf}/include/SDL"
  ]
  ++ stdenv.lib.optionals (stdenv.hostPlatform.isLinux) [
    "-I${gtk2.dev}/include/gtk-2.0"
    "-I${glib.dev}/include/glib-2.0"
  ]);
```

---

We can try when the darwin builder comes back up